lib/zipper.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
         });
         return pairs.concat(noParams);
     },
-    generateImportsTxt: function (pairs, indent) {
+    generateImportsTxt: function (pairs, indent, groupPackages) {
```

Comment:
I have changed the default behaviour of zipper, but since it now is an option, and the default value is true, it is on by default.